src/components/UserImpact.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react'
```

Comment:
Please add tests for these changes or chat with me if you don't feel it's worth the effort.